bootstrap/velocity/src/main/java/org/geysermc/platform/velocity/GeyserVelocityPlugin.java

```diff
@@ -68,11 +68,10 @@
     private IGeyserPingPassthrough geyserPingPassthrough;
 
     private GeyserConnector connector;
+    private final File configDir = new File("plugins/" + GeyserConnector.NAME + "-Velocity/");
```

Comment:
I think it would be better to store these as a Path

Reply:
^

Reply:
I'll update that, but it will require a lot of toFile()'s for that plugin.